Fix possibly undefined cpu variable when !CONFIG_SMT_NICE.

Patch courtesy of Alfred Chen.

---
 kernel/sched/bfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-4.7-ck5/kernel/sched/bfs.c
===================================================================
--- linux-4.7-ck5.orig/kernel/sched/bfs.c	2016-09-23 08:32:56.347748787 +1000
+++ linux-4.7-ck5/kernel/sched/bfs.c	2016-09-23 08:32:56.345748800 +1000
@@ -1405,8 +1405,8 @@ static void try_preempt(struct task_stru
 	}
 
 	if (likely(highest_prio_rq)) {
-#ifdef CONFIG_SMT_NICE
 		cpu = cpu_of(highest_prio_rq);
+#ifdef CONFIG_SMT_NICE
 		if (!smt_should_schedule(p, cpu))
 			return;
 #endif
